h2o-docs/src/front/index.html

```diff
@@ -98,7 +98,7 @@ <h2 class="wow fadeInUp" data-wow-delay=".3s"><span class="bg-black text-yellow"
       <div class="column-wrapper">
         <div class="col-two left text-left">
           <h4 class="underline-yellow wow wefadeInUp">H2O-3</h4>
-          <h5 style="padding-top: 10px;">Version 3.30.0.5</h5>
+          <h5 style="padding-top: 10px;">Version 3.38.0.3</h5>
```

Comment:
Why we need to change this? It should be automatically generated in production

Reply:
When I go to the most recent index, http://h2o-release.s3.amazonaws.com/h2o/rel-zygmund/2/docs-website/index.html, it says 3.30.0.5 for me. Is this just a cache issue on my end?

Reply:
right, looks like a bug in the release process, we should have a placeholder here instead replaced by release script.

Reply:
I'd keep @hannah-tillman update for now, and create a ticket to fix this

Reply:
👍